dockerfiles/Dockerfile-rosetta

```diff
@@ -190,8 +193,8 @@ RUN opam install \
   re \
   result \
   stringext) \
-  && opam switch import src/opam.export \
-  && sudo chmod -R u+rw ~/.opam
+  && opam switch import src/opam.export
+#  && sudo chmod -R u+rw ~/.opam
```

Comment:
Nice! This is unnecessary?

Reply:
I pulled it from a script because of a specific pin, and either the bug is resolved or it was digestif and we're not using the pin anymore.

I have not seen builds fail since removing it, and ~/.opam contains so many files that this was taking almost 3 minutes to execute.

If the original issue resurfaces I can make it much more selective as to what we chmod, but chmod -R for that whole dir is too costly.